trino/trino.go

```diff
@@ -104,6 +104,7 @@ var (
 
 const (
 	trinoHeaderPrefix = `X-Trino-`
+	trinoQueryPrefix  = `X-Query-`
```

Comment:
We already recognize `X-Trino-User` in named parameters. I don't you should introduce a new prefix.

Reply:
Ok, then would `X-Trino-Progress-Callback` and `X-Trino-Progress-Callback-Period` be ok?